packages/components/.storybook/web/main.ts

```diff
@@ -21,9 +20,6 @@ module.exports = {
   },
   framework: '@storybook/react-webpack5',
   options: { builder: { useSWC: true } }, // improves build performance
-  typescript: {
-    reactDocgen: 'react-docgen',
```

Comment:
With Storybook 7, TypeScript is natively supported so this shouldn't be needed. This was screwing up Button's Doc page Controls panel (possibly due to the way we're currently using an enum for setting buttonType values) and I didn't notice any issues removing it.